src/bootstrap.js

```diff
@@ -172,8 +181,19 @@ var mofofuraco =
         try
         {
           let paypalData = JSON.parse(xhr.responseText);
+          let previousAmount = mofofuraco.amount;
           mofofuraco.amount = Math.round(paypalData.sum);
           mofofuraco.updateLabelWithAmount();
+          if ( Preferences.get("extensions.mofofuraco.moneySound") === true)
+          {
+            // if the modulous of the previous amount and 100000 is less than
+            // the difference between the previous amount and new amount, we've
+            // raised another $100,000!
+            if ((previousAmount % 100000) < (mofofuraco.amount - previousAmount))
```

Comment:
1. The amount of money spent in the last time interval isn't a variable in this check, but only previousAmount and the current amount. So Math.floor for both numbers % 100000.
2. One has to check for previousAmount !== 0 because that is the initial startup value which would cause Firefox to always play the song when it launches.